python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/vlan_interfaces.schema.yml

```diff
@@ -644,6 +644,12 @@ keys:
                 input:
                   type: str
                   description: Name of policy-map used for policy based routing.
+        traffic_policy_input:
```

Comment:
please use the same model as for port_channel and ethernet interfaces so we (and our users!) keep sane :)

```
        traffic_policy:
          type: dict
          keys:
            input:
              type: str
              description: Ingress traffic policy.
            output:
              type: str
              description: Egress traffic policy.
```

Also for documentation lets align the documentation and remove the two columns you have added and add the Vlan interfaces to the table "Traffic-policy interfaces" in `traffic-policies.j2`